_posts/2017-06-26-http2.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: HTTP2 on MDN
+author: Stephanie Hobson
+excerpt_separator: <!--more-->
+---
+
+We enabled HTTP/2 on MDN's CDN.
+
+We didn't do anything to optimize for HTTP/2, we just enabled it.
+
+We're seeing performance improvements.
+
+<!--more-->
+
+Why is this worth a blog post?
+===
+
+While doing research to see if turning it on without doing any optimizations was
+a good idea I read things like:
+
+> "It also means that all of those HTTP1 performance techniques are harmful.
+> They will make a HTTP2 website slower, not faster - don't use them." -
+> [HTTP2 for front-end web developers](https://mattwilcox.net/web-development/http2-for-front-end-web-developers)
+
+And:
+
+> "However, many of the things you think of as being best practices can be
+> detrimental to performance under HTTP/2." - [Getting Ready For HTTP2: A Guide
+> For Web Designers And Developers](https://www.smashingmagazine.com/2016/02/getting-ready-for-http2/)
+
+Which suggest that enabling HTTP/2 on a site optimized for HTTP/1.1 could result
+in a slower site.
+
+A better way to interpret those quotes is:
+
+> If you optimize for HTTP/1.1 and turn on HTTP/2 your site will not be as fast
+> as it could be - but it might still be faster!
+
+On MDN we concatenate a lot of our files but we don't concatenate all of them.
+For example, our article pages we have 9 different files coming from our CDN. I
```

Comment:
I think this should be "**on** our article pages we have 9 different files" or "our article pages ~~we~~ have 9 different files..."